app/utils/EitherTUtils.scala

```diff
@@ -19,12 +19,20 @@ package utils
 import cats.Functor
 import cats.data.EitherT
 import models.mongo.ServiceError
+import play.api.mvc.Result
+
+import scala.concurrent.{ExecutionContext, Future}
 
 object EitherTUtils {
 
-  implicit class EitherTOps[F[_]: Functor, A <: ServiceError, B](value: EitherT[F, A, B]) {
+  implicit class CasterOps[F[_]: Functor, A <: ServiceError, B](value: EitherT[F, A, B]) {
```

Comment:
CasterOps? Why not EitherTOps, it was wrapper around EitherT

Reply:
Well I wanted to add another extension but my types didn't meet the A <: ServiceError constraint. I therefore needed to add another class, which is also ops on EitherT but couldn't have the same name so needed to differentiate them somehow. I thought `CasterOps` made sense as it is conducting casting operations on the EitherT. What do you think?